assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -110,9 +50,77 @@ const cartTotals = {
 /**
  * Component that renders the Cart block when user has something in cart aka "full".
  */
-const Cart = () => {
-	const currency = getCurrencyFromPriceResponse( cartTotals );
-	const totalRowsConfig = getTotalRowsConfig( cartTotals );
+const Cart = ( { isShippingCalculatorEnabled, isShippingCostHidden } ) => {
+	/**
+	 * Given an API response with cart totals, generates an array of rows to display in the Cart block.
+	 *
+	 * @param {Object} cartTotals - Cart totals data as provided by the API.
+	 * @return {Object[]} Values to display in the cart block.
+	 */
+	const getTotalRowsConfig = ( cartTotals ) => {
+		const totalItems = parseInt( cartTotals.total_items, 10 );
+		const totalItemsTax = parseInt( cartTotals.total_items_tax, 10 );
+		const totalRowsConfig = [
+			{
+				label: __( 'List items:', 'woo-gutenberg-products-block' ),
+				value: DISPLAY_PRICES_INCLUDING_TAXES
+					? totalItems + totalItemsTax
+					: totalItems,
+			},
+		];
+		const totalFees = parseInt( cartTotals.total_fees, 10 );
+		if ( totalFees > 0 ) {
+			const totalFeesTax = parseInt( cartTotals.total_fees_tax, 10 );
+			totalRowsConfig.push( {
+				label: __( 'Fees:', 'woo-gutenberg-products-block' ),
+				value: DISPLAY_PRICES_INCLUDING_TAXES
+					? totalFees + totalFeesTax
+					: totalFees,
+			} );
+		}
+		const totalDiscount = parseInt( cartTotals.total_discount, 10 );
+		if ( totalDiscount > 0 ) {
+			const totalDiscountTax = parseInt(
```

Comment:
Should this be in this PR? Looks unrelated to shipping options to me :)

Reply:
This was here when I started editing, it only shows up as edited since I moved the function from outside the component to inside it, the changes I made in `assets/js/blocks/cart-checkout/cart/full-cart/index.js` are adding two props, and adding a condition to certain items according to those props.